Frends.FixedWidthFlatFile.ConvertToJSON/Frends.FixedWidthFlatFile.ConvertToJSON/Definitions/Result.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace Frends.FixedWidthFlatFile.ConvertToJSON.Definitions
+{
+    public class Result
+    {
+        /// <summary>
+        /// Column name. If input data contains Header row and value is left empty, header value is used as name.
+        /// </summary>
+        [DisplayFormat(DataFormatString = "Text")]
```

Comment:
This line can be removed, since this is the result of the task and user doesn't give any input to this property.